.github/workflows/tests.yml

```diff
@@ -217,6 +217,11 @@ jobs:
           # TODO: remove the line below once set in the runner .env file
           echo 'GITHUB_ACTIONS_HOMEBREW_SELF_HOSTED=1' >> $GITHUB_ENV
 
+      - name: Set environment variables
+        if: matrix.container == 'homebrew/debian7:latest'
+        run: |
+          echo 'HOMEBREW_CURL_PATH=/opt/bin/curl' >> $GITHUB_ENV
```

Comment:
We should handle this in `test-bot`. Doing this here probably won't work anyway, since we explicitly set `HOMEBREW_CURL_PATH` in `lib/test_bot.rb`.

Reply:
I did not know we were doing this.

I was also thinking baking in the ENV variable into the docker image, but I want to hard-code this here first to test this.

Let's first discuss the next steps in test bot then: https://github.com/Homebrew/homebrew-test-bot/pull/786

Reply:
Agreed.